en/CONTRIBUTE.md

```diff
@@ -9,12 +9,12 @@ We appreciate your help and contributions!
 ## Getting started with contribution
 
 * Get started with the WSO2 API Manager by accessing the [documentation landing](https://apim.docs.wso2.com/en/latest/). Start trying out our quick start guides and tutorials available in the Get Started and Learn sections.
-* Submit issues and contribute content. Go to the **Issues** tab of this GitHub repo and click the New Issue button to file a bug report.
+* Submit issues and contribute content. Go to the **Issues** tab of this GitHub repo and click the **New Issue** button to file a bug report.
 * Fix issues: Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
 
 ## Filing issues
 
-If you are unsure whether you have found a bug, please consider searching existing issues in GitHub and asking in dev@wso2.com or on [Stack Overflow](https://stackoverflow.com/tags/wso2).
+If you are unsure whether you have found a bug, please consider searching existing issues in GitHub and asking on [Stack Overflow](https://stackoverflow.com/tags/wso2).
```

Comment:
Better to tell them that we can reach out on Slack as well.

Reply:
Fixed.